lifecycle-operator/config/crd/kustomization.yaml

```diff
@@ -14,32 +14,34 @@ resources:
   - bases/options.keptn.sh_keptnconfigs.yaml
   - bases/lifecycle.keptn.sh_keptnappcreationrequests.yaml
 # +kubebuilder:scaffold:crdkustomizeresource
-
-patchesStrategicMerge:
-  # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
-  # patches here are for enabling the conversion webhook for each CRD
-  # - patches/webhook_in_lifecycle_keptntaskdefinitions.yaml
-  # - patches/webhook_in_lifecycle_keptntasks.yaml
-  - patches/webhook_in_lifecycle_keptnapps.yaml
-  # - patches/webhook_in_lifecycle_keptnworkloads.yaml
-  - patches/webhook_in_lifecycle_keptnworkloadinstances.yaml
-  - patches/webhook_in_lifecycle_keptnappversions.yaml
-  # - patches/webhook_in_lifecycle_keptnevaluationdefinitions.yaml
-  - patches/webhook_in_lifecycle_keptnevaluationproviders.yaml
-  # - patches/webhook_in_lifecycle_keptnevaluations.yaml
-  # - patches/webhook_in_options_keptnconfigs.yaml
-  # - patches/webhook_in_lifecycle_keptnapps.yaml
-  # - patches/webhook_in_lifecycle_keptnappversions.yaml
-  # - patches/webhook_in_lifecycle_keptnworkloadinstances.yaml
-  # - patches/webhook_in_lifecycle_keptnevaluationproviders.yaml
-  # - patches/webhook_in_keptnappcreationrequests.yaml
-  # +kubebuilder:scaffold:crdkustomizewebhookpatch
-
 # the following config is for teaching kustomize how to do kustomization for CRDs.
 configurations:
   - kustomizeconfig.yaml
 
-commonLabels:
-  crdGroup: lifecycle.keptn.sh
-  app.kubernetes.io/part-of: keptn-lifecycle-toolkit
-  keptn.sh/inject-cert: "true"
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+labels:
+  - includeSelectors: true
+    pairs:
+      app.kubernetes.io/part-of: keptn-lifecycle-toolkit
+      crdGroup: lifecycle.keptn.sh
+      keptn.sh/inject-cert: "true"
+patches:
+  - path: patches/webhook_in_lifecycle_keptnapps.yaml
+  - path: patches/webhook_in_lifecycle_keptnworkloadinstances.yaml
+  - path: patches/webhook_in_lifecycle_keptnappversions.yaml
+  - path: patches/webhook_in_lifecycle_keptnevaluationproviders.yaml
+# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
+# patches here are for enabling the conversion webhook for each CRD
+# - path: patches/webhook_in_lifecycle_keptntaskdefinitions.yaml
+# - path: patches/webhook_in_lifecycle_keptntasks.yaml
+# - path: patches/webhook_in_lifecycle_keptnworkloads.yaml
+# - path: patches/webhook_in_lifecycle_keptnevaluationdefinitions.yaml
+# - path: patches/webhook_in_lifecycle_keptnevaluations.yaml
+# - path: patches/webhook_in_options_keptnconfigs.yaml
+# - path: patches/webhook_in_lifecycle_keptnapps.yaml
+# - path: patches/webhook_in_lifecycle_keptnappversions.yaml
+# - path: patches/webhook_in_lifecycle_keptnworkloadinstances.yaml
+# - path: patches/webhook_in_lifecycle_keptnevaluationproviders.yaml
```

Comment:
Please do not add patches in the comments which are already uncommented
```suggestion
```

Reply:
Oh, sorry I havent seen it.